ultra/README.md

```diff
@@ -32,7 +32,7 @@ For a longer introduction to ULTRA, including its purpose, concepts, and benchma
 If you use ULTRA in your research, you can use the following citation.
 ```
 @misc{elsayed20_ultra,
-  author          = {Elsayed, Mohamed and Hassanzadeh, Kimia and Nguyen, Nhat M,
+  author          = {Elsayed, Mohamed and Hassanzadeh, Kimia and Nguyen, Nhat M
```

Comment:
```suggestion
  author          = {Elsayed, Mohamed and Hassanzadeh, Kimia and Nguyen, Nhat M.
```
Whoops, I think this should have been a period since it's an initial.

Reply:
Oh yes, good call!